Add Ibeji, Agemo, and Freyja in-vehicle stack blueprint image workflow

ladatz opened the pull request:
These github workflows checks out the eclipse-chariott/Agemo repository  and eclipse-ibeji/ibeji repository and builds images with the configuration and settings needed to run in the integrated mode with the other workloads in the software-orchestration blueprint. It then pushes the resulting images to the blueprint GitHub container registry. The resulting images include:

- eclipse-agemo pub-sub-service
- An eclipse mosquitto mqtt broker using a custom config with agemo
- eclipse-ibeji invehicle digital twin service
- eclipse-freyja a digital twin cloud-synchronization application with a mocked cloud component and integration with ibeji

ashbeitz reviewed (approved):
Approved

ladatz commented:
@eriksven Can we please "squash and merge" this one once it has been reviewed? I ended up with a lot of commits here. I'll make sure the next one is cleaner

eriksven commented on .github/workflows/containerize-invehicle-stack-agemo.yml:
```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: MIT
```
The specified license in the LICENSE file in the root directory of this repository is Apache 2.0. Do you really want to license the workflows under another license here? 

In addition, the Eclipse Project handbook proposes to include a full copyright header in addition to the License Identifier. See #16 for more details.

ladatz replied:
I am waiting for some confirmation from our side about the license question

eriksven replied:
@ladatz any updates on the licensing topics here?

ladatz replied:
@eriksven As far as I know, we are still waiting on our open source office team which is reviewing contribution process. I heard we had made some progress and I will update this once I have the green light

ladatz replied:
@eriksven We are good to merge this. I have updated the license to Apache 2, to be consistent with the rest of the blueprint

eriksven commented:
The workflows look good to me. I added a comment to add a full copyright header. Also there is a merge conflict with the chariott workflow. Can you resolve this? 

For the merge I will use "squash and merge" this time.

ladatz commented:
> The workflows look good to me. I added a comment to add a full copyright header. Also there is a merge conflict with the chariott workflow. Can you resolve this?
> 
> For the merge I will use "squash and merge" this time.

I am looking for some confirmation from my team about the license question. I also cleaned up the history of this PR and added the Freyja workflow as well.

inf17101 commented on .github/workflows/containerize-invehicle-stack-agemo.yml:
```diff
@@ -0,0 +1,51 @@
+        with:
+          repository: eclipse-chariott/Agemo
+          submodules: recursive
+          ref: 0.1.2
+
+      - name: Build and push multi-platform Docker image for Mosquitto image with Agemo config
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: Dockerfile.mosquitto.multi
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ghcr.io/eclipse-sdv-blueprints/software-orchestration/eclipse-mosquitto/mqtt-broker-agemo-config:0.1.2
```
I am thinking about, shall we define the version tag one times e.g. as Github action env variable to ensure to reduce possibility of making mistakes. Someone might forget to adjust the image version tag when checking out another agemo version. To define this once would keep it automatically in sync. We could do this for all image builds as well, if you like the idea.

inf17101 replied:
I have seen that you have accepted it and pushed it, but unfortunately I am not able to close the conversation.

inf17101 commented on .github/workflows/containerize-sample-workloads.yaml.yml:
Why is it .yaml.yml?

ladatz replied:
This was a typo and the file has been removed from this PR altogether (that workflow will go in with the sample code).

eriksven commented:
@ladatz I missed to see that you already updated the license headers in this PR and merged #20 before now. Most likely as a result, there is now a merge conflict with the containerize-invehicle-stack-chariott.yml file. Can you resolve this?

ladatz commented:
> @ladatz I missed to see that you already updated the license headers in this PR and merged #20 before now. Most likely as a result, there is now a merge conflict with the containerize-invehicle-stack-chariott.yml file. Can you resolve this?



> @ladatz I missed to see that you already updated the license headers in this PR and merged #20 before now. Most likely as a result, there is now a merge conflict with the containerize-invehicle-stack-chariott.yml file. Can you resolve this?

@eriksven No problem, I have resolved the merge conflict now.
